scala_proto/scala_proto.bzl

```diff
@@ -320,12 +320,15 @@ def _colon_paths(data):
 
 def _gen_proto_srcjar_impl(ctx):
     acc_imports = depset()
+    transitive_proto_path_flags = depset()
 
     proto_deps, jvm_deps = [], []
     for target in ctx.attr.deps:
         if hasattr(target, 'proto'):
             proto_deps.append(target)
             acc_imports += target.proto.transitive_sources
+            if hasattr(target.proto, 'transitive_proto_path_flags'):
```

Comment:
we should have either an issue or a code comment on these version compatibility check things so we don't lose track.

Like after version `0.11` is the oldest supported version, remove this check or something. What do you think?

I'm worried about the code building up layers of these checks we don't recall why the exist.